python/test_support/sql/streaming/apply_in_pandas_with_state/random_failure/input/test-00000.txt

```diff
@@ -0,0 +1,100 @@
+non
```

Comment:
Can we avoid adding these files? I try to avoid this in general to make the test case self-contained and more readable.

Reply:
I just changed both tests to create a dataset files before running a test; neither input files nor golden file is needed.